### PR TITLE
Move search bar and download button to app bar

### LINKED ISSUE #948: UX tweaks
**Describe the bug**

- Tooltip icon should be in the search field 

- Move search bar up to nav bar
- Add download button
- Move pagination up
- Change type face for table rows (not headers) and pagination to Inter

**Screenshots**
https://screenshot.googleplex.com/DNCfDQAirMNvecn

See mocks here for details:
https://www.figma.com/file/TqqzDnXF4KTBAGUchpt4mQ/Global.Health-Marketing-Website?node-id=278%3A2

### PR DESCRIPTION
**Pull request by allysonjp715**
Fixes #948

Screen Shot 2020-09-11 at 1 32 26 PM

Screen Shot 2020-09-11 at 1 32 45 PM

### CODE REVIEW

File: verification/curator-service/ui/src/components/App.tsx [hunk 1]
```diff
@@ -267,8 +276,12 @@ export default function App(): JSX.Element {
         setCreateNewButtonAnchorEl,
     ] = useState<Element | null>();
     const [selectedMenuIndex, setSelectedMenuIndex] = React.useState<number>();
+    const [linelistSearchQuery, setLinelistSearchQuery] = React.useState<
+        string
```
Comment (attwad) [inline verification/curator-service/ui/src/components/App.tsx:280]: you don't need the <string> here if you provide a default value from which the type can be inferred

File: verification/curator-service/ui/src/components/LinelistTable.tsx [hunk 2]
```diff
@@ -82,13 +81,13 @@ interface LocationState {
...
     editedCaseIds: string[];
     bulkMessage: string;
-    searchQuery: string;
 }
 
 interface Props
     extends RouteComponentProps<never, never, LocationState>,
         WithStyles<typeof styles> {
     user: User;
+    search: string;
```
Comment (attwad) [inline verification/curator-service/ui/src/components/LinelistTable.tsx:90]: `search?` would make you able to not specify it everywhere when instantiating the component (but perhaps involves a bunch of checks to be done here as well so might not be beneficial in the end...)

File: verification/curator-service/ui/src/components/LinelistTable.tsx [hunk 3]
```diff
@@ -165,7 +156,7 @@ const StyledSearchTextField = withStyles({
     },
 })(TextField);
 
-function SearchBar(props: {
+export function SearchBar(props: {
```
Comment (attwad) [inline verification/curator-service/ui/src/components/LinelistTable.tsx:159]: Now it feels like this would live better in its own file, up to you.

**Review by attwad — APPROVED** [review]

File: verification/curator-service/ui/src/components/App.tsx (hunk 1 above)
Comment (allysonjp715) [reply verification/curator-service/ui/src/components/App.tsx:280]: Done.

File: verification/curator-service/ui/src/components/LinelistTable.tsx (hunk 2 above)
Comment (allysonjp715) [reply verification/curator-service/ui/src/components/LinelistTable.tsx:90]: Yea, I think that'd only really be useful for tests, where it's not too difficult to add it. Don't think it's worth the extra checks in non-test code.

File: verification/curator-service/ui/src/components/LinelistTable.tsx (hunk 3 above)
Comment (allysonjp715) [reply verification/curator-service/ui/src/components/LinelistTable.tsx]: Yea agreed. Moved.

**Pull request merged**